user_guide_src/source/general/configuration.rst

```diff
@@ -93,7 +93,7 @@ Creating Dotenv File
 
 CodeIgniter expects the **.env** file to be at the root of your project alongside the
 **app** directories. There is a template file distributed with CodeIgniter that's
-located at the project root named **env** (Notice there's no dot (``.``) at the start?).
+located at the project root named **env** (Notice there's no dot (``.``) at the start.).
```

Comment:
I don't think this is a typo.

Reply:
You are right. Google translate says so too. It means 'Did you notice that there is no dot at the start?'. Thanks.